### PR TITLE
Changed folder structure and made packages for specific use case.

### PR DESCRIPTION
**Pull request by srrathi**
- [ ] To add tests
- [x] Proper commenting before every function.
- [ ] To add a go linter

### CODE REVIEW

File: ui/utils.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+	// load .env file
+	err := godotenv.Load(".env")
+
+	if err != nil {
+		log.Fatalf("Error loading .env file")
+	}
+
+	return os.Getenv(key)
+}
```
Comment (debdutdeb) [inline ui/utils.go]: Load the file once in `init` - then maybe get each key just once and export. Like,
```go
func init() {
    if err := godotenv.Load(".env"); err != nil {
        // handle error
    }
}
var SITE_URL = os.Getenv("DEV_SERVER_URL")
var EMAIL = os.Getenv("EMAIL")
var PASS = os.Getenv("PASS")
```
now just reuse those.

File: ui/utils.go [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+func getServerUrl() string {
+	DEV_SERVER_URL := goDotEnvVariable("DEV_SERVER_URL")
+	return DEV_SERVER_URL
+}
```
Comment (debdutdeb) [inline ui/utils.go]: As said above you don't need this.

**Review by debdutdeb — CHANGES_REQUESTED** [review]
Just a quick review on a sunday. As soon as I actually run it locally, I'll leave another one.

File: ui/utils.go (hunk 1 above)
Comment (srrathi) [reply ui/utils.go]: yes it shouldn't be loaded everytime I call the function. I'll update it.

**Comment by srrathi** [comment]
@debdutdeb I think we can merge it for now. If found any bugs and error we can open issue for it.

**Comment by debdutdeb** [comment]
Sounds good @srrathi :)

**Review by debdutdeb — APPROVED** [review]

**Comment by debdutdeb** [comment]
Ah I can't merge this, @geekgonecrazy ping

**Comment by srrathi** [comment]
> Ah I can't merge this, @geekgonecrazy ping

@debdutdeb I can merge it. But may be @geekgonecrazy could give a quick feedback, we can merge after it. Wys

**Pull request merged**